lib/model/KnowledgePanel.dart

```diff
@@ -60,65 +40,57 @@ enum Grade {
 enum Evaluation {
   @JsonValue('good')
   GOOD,
-  @JsonValue('neutral')
-  NEUTRAL,
+  @JsonValue('average')
+  AVERAGE,
   @JsonValue('bad')
   BAD,
   UNKNOWN,
```

Comment:
why no @JsonValue('unknown') ?

Reply:
right: environment/packaging_recycling.tt.json:                    "evaluation": "neutral",

Reply:
what's the difference between "average" and "neutral"? Can't we collapse them into one ? 


Unknown: the object gets a value of enum type UNKNOWN when the json parser encounters a value from the server that isn't mappable to anything else in this enum (specifically if server responds with something other than Good, bad, average, the variable is assigned UNKNOWN).